starters/angular-apollo-tailwind/.gitignore

```diff
@@ -40,3 +40,7 @@ testem.log
 # System files
 .DS_Store
 Thumbs.db
+
+# Auto-Generated
+
+documentation.json
```

Comment:
Do we still need to be generating this? It's in the package.json scripts but I wonder if we really need it.

Reply:
I guess I don't understand why this is in the kits. Does anyone know? @morgnism @BrettZeidler

Reply:
yeah not sure why specifically -- I think it was added before I was brought into the project

would be best to .gitignore it though I'd say